### ISSUE TITLE
Endurance Potion Effect

### ISSUE THREAD

**Issue by WACriminalG2** [Tictim/Paraglider#26]
Would be really cool if there was a potion effect you could have that slowed your rate of stamina consumption. Due to the way paragliding works, you can't eat or drink anything while gliding like in Breath of the Wild, so the early-game role filled by stamina-restoring foods isn't possible unless you eat REALLY FAST while falling, LOL. A potion effect you could apply to yourself prior to starting your glide would fix that.

**Comment by Tictim** [comment]
Yeah, there was some internal discussions about method of mitigate stamina consumption, like enchantments or some sort. But even with potions I think it would be somewhat janky and hard to use, I assume.

Also there's a problem with potion system itself. Even if you loot brewing stand somewhere in overworld, burning exotic and annoying to get ingredients to get 5 seconds worth of stamina is not a very good deal.

**Comment by Tictim** [comment]
Implemented in 21.x, as Energizing and Enduring Elixirs.
